src/main/java/com/exadel/discountwebapp/location/controller/LocationController.java

```diff
@@ -0,0 +1,48 @@
+package com.exadel.discountwebapp.location.controller;
+
+import com.exadel.discountwebapp.location.entity.Location;
+import com.exadel.discountwebapp.location.service.LocationService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/location")
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class LocationController {
+    LocationService locationService;
+
+    @GetMapping("/")
+    private List<Location> getAllLocations() {
+        return locationService.findAll();
+    }
+
+    @GetMapping("/{county}")
+    private List<Location> getALlLocationByCounty(@PathVariable String county) {
```

Comment:
You can use @GetMapping("/{county}") or @GetMapping("/{city}") in same controller

Reply:
How do you mean to do this. We have 2 separate services wich return list of Locations. 

If you mean @GetMapping("/{county}/{city}")

then we have merge list of Location found by county and by city. Or you have another idea? 
I think 2 separate API better.